etcdserver/api/rafthttp/http.go

```diff
@@ -287,6 +289,7 @@ func (h *snapshotHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			plog.Error(msg)
 		}
 		http.Error(w, msg, http.StatusInternalServerError)
+		snapshotReceiveInflights.WithLabelValues(from).Dec()
```

Comment:
nit: Would be great if we could somehow structure the code blocks we only need to make one call to `Dec()`, ideally in a defer.  The code looks correct, but I worry we'll end up miscounting at some point...